lib/tether/tether.go

```diff
@@ -236,12 +236,17 @@ func (t *tether) setNetworks() error {
 
 func (t *tether) setMounts() error {
 	for k, v := range t.config.Mounts {
-		if v.Source.Scheme != "label" {
+		switch v.Source.Scheme {
+		case "label":
+			// this could block indefinitely while waiting for a volume to present
+			t.ops.MountLabel(context.Background(), v.Source.Path, v.Path)
+
+		case "nfs":
+			t.ops.MountFileSystem(context.Background(), v.Source, v.Path, v.Mode)
```

Comment:
`MountFileSystem(ctx context.Context, source url.URL, target string, mountOptions string)`

That means you're passing the `Mode` as the mountOptions?  I thought it needed to be encoded or something.  Maybe adding a comment in the MountFileSystem call for what form the mountOptions take is what's missing.